clang/lib/CIR/CodeGen/CIRGenExprConst.cpp

```diff
@@ -1059,7 +1059,7 @@ class ConstantLValueEmitter
 
     llvm::SmallVector<mlir::Attribute, 3> Indices;
     for (auto I : Idx) {
-      auto Attr = mlir::cir::IntAttr::get(CGM.getBuilder().getSInt64Ty(), I);
+      auto Attr = CGM.getBuilder().getIndexAttr(I);
```

Comment:
Or maybe use `getI32IntegerAttr` to match `GEPArg` in LLVM IR dialect.

Reply:
I prefer we do not use IndexAttr until we see a real use case for it, `getI32IntegerAttr` would be great.